app/scripts/controllers/transactions/index.test.js

```diff
@@ -2200,14 +2200,17 @@ describe('Transaction Controller', function () {
         estimateUsed: '0x009',
       });
 
-      try {
-        txController.updateTransactionGasFees('4', { maxFeePerGas: '0x0088' });
-      } catch (e) {
-        assert.equal(
-          e.message,
-          'Cannot call updateTransactionGasFees on a transaction that is not in an unapproved state',
-        );
-      }
+      assert.throws(
+        () =>
+          txController.updateTransactionGasFees('4', {
+            maxFeePerGas: '0x0088',
+          }),
+        Error,
+        'Cannot call updateTransactionGasFees on a transaction that is not in an unapproved state',
```

Comment:
[nit] could we make this error `Cannot call updateTransactionGasFees on a transaction that is in an approved state` 
(not a double negative)

Reply:
@adonesky1 I agree these messages were confusing. Thanks for calling this out!

Currently, the transactions can have these statuses: 
``` 
  UNAPPROVED: 'unapproved',
  APPROVED: 'approved',
  REJECTED: 'rejected',
  SIGNED: 'signed',
  SUBMITTED: 'submitted',
  FAILED: 'failed',
  DROPPED: 'dropped',
  CONFIRMED: 'confirmed',
  PENDING: 'pending',
```

The calls should be blocked for other transactions that are not `approved`, like `dropped`. I updated the related function name and the error messages to hopefully add some clarity.